docker/Dockerfile.debug

```diff
@@ -1,22 +1,26 @@
 FROM ubuntu:xenial
+# Do not add more stuff to this list that isn't small or critically useful.
+# If you occasionally need something on the container do
+# sudo /usr/local/bin/proxy-redirection-clear
+# sudo apt-get update && apt-get whichever
+# sudo /usr/local/bin/proxy-redirection-restore
 RUN apt-get update && \
-    apt-get install -y \
-    curl \
-    iptables \
-    iproute2 \
-    iputils-ping \
-    dnsutils \
-    netcat \
-    tcpdump \
-    net-tools \
-    libc6-dbg gdb valgrind \
-    vim \
-    nano \
-    emacs \
-    wrk \
-    lsof \
-    busybox \
-    sudo
+    apt-get install --no-install-recommends -y \
+      curl \
+      iptables \
+      iproute2 \
+      iputils-ping \
+      dnsutils \
+      netcat \
+      tcpdump \
+      net-tools \
+      libc6-dbg gdb \
+      elvis-tiny \
```

Comment:
this is like vim but 94k instead of megabytes

and anyone can sudo apt-get their favorite editor if they really have to edit things in the container and don't know `ESC:wq` for save and quit and `i` to insert text